src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -421,8 +428,8 @@ public NoSyncWorkspaceConfigurationService()
                 sourceTexts = values[i];
 
                 // make a change to the project to force a change between the local and oop solutions.
-                var currentProject = project.AddDocument("X.cs", SourceText.From("// " + i)).Project;
-                Assert.True(localWorkspace.SetCurrentSolution(_ => currentProject.Solution, WorkspaceChangeKind.SolutionChanged));
+
```

Comment:
```suggestion
```
Not sure if the formatting analyzer might complain here or not.

Reply:
it seems happy :)